### PR TITLE
Switch to openshift/api CRD manifest

### PR DESCRIPTION
**Pull request by sttts**

### CODE REVIEW

**Comment by soltysh** [comment]
/cc @adambkaplan 
please use this pattern when updating ocm-o to use latest openshift/api

File: Dockerfile.rhel7 [hunk 1]
```diff
@@ -6,4 +6,5 @@ RUN go build ./cmd/cluster-openshift-apiserver-operator
 FROM registry.svc.ci.openshift.org/ocp/4.2:base
 COPY --from=builder /go/src/github.com/openshift/cluster-openshift-apiserver-operator/cluster-openshift-apiserver-operator /usr/bin/
 COPY manifests /manifests
+COPY vendor/github.com/openshift/api/operator/v1/*_openshift-apiserver-operator_*,crd.yaml manifests/
```
Comment (sttts) [inline Dockerfile.rhel7]: grrr, typo

**Pull request merged**